### ISSUE TITLE
FileNotFoundException thrown by MockFileSystem and by real System.IO doesn't match

### ISSUE THREAD

**Issue by LatteAno** [System-IO-Abstractions/System.IO.Abstractions#424]
This unit test fails:

```
[TestMethod]
public void TestFileNotFoundExceptionContent() {
	var expectedMsg = "";
	var expectedFileName = "";
	var actualMsg = "";
	var actualFileName = "";

	var absentFileNameFullPath = @"c:\you surely don't have such file.hope-so";
	try {
		File.ReadAllText(absentFileNameFullPath);
	} catch (FileNotFoundException e) {
		expectedMsg = e.Message;
		expectedFileName = e.FileName;
	}

	try {
		_fileSystemMock.File.ReadAllText(absentFileNameFullPath);
	} catch (FileNotFoundException e) {
		actualMsg = e.Message;
		actualFileName = e.FileName;
	}
	
	// Expected:	@"Could not find file 'c:\you surely don't have such file.hope-so'."
	// Actual:	@"Can't find c:\you surely don't have such file.hope-so"
	Assert.AreEqual(expectedMsg, actualMsg);
	// Expected:	@"c:\you surely don't have such file.hope-so"
	// Actual:	null
	Assert.AreEqual(expectedFileName, actualFileName);
}
```

**Comment by fgreinacher** [comment]
Thanks, this is addressed by #427

**Comment by fgreinacher** [comment]
Fixed in https://www.nuget.org/packages/System.IO.Abstractions/3.0.10
